types/sphere-engine-browser/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// Type definitions for the non-npm package sphere-engine-browser 5.5
+// Project: https://github.com/fatcerberus/miniSphere
+// Definitions by: Fat Cerberus <https://github.com/fatcerberus>, Eggbertx <https://github.com/eggbertx>
```

Comment:
Did @fatcerberus work on these definitions with you, or just the source library? (Also their GitHub display name is Bruce Pascoe, not Fat Cerberus 😄)

Reply:
He created the miniSphere engine itself and sphere2-api.d.ts (with my modifications to make it dtlint compatible), and I created sphere1-api.d.ts based on the original Sphere engine's API, which miniSphere is mostly backwards compatible with. And I put "Fat Cerberus" as the name in index.d.ts since that's what he put in sphere2-api.d.ts

Reply:
Wait, what are sphere2-api.d.ts and sphere1-api.d.ts